angular-ngrx-scss/src/app/home/top-repositories/top-repositories.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { ProfileState } from 'src/app/state/profile/profile.state';
+import { selectTopRepos, selectUserLoginName } from 'src/app/state/user';
+import languageColors from 'src/assets/language-colors.json';
+
+@Component({
+  selector: 'app-top-repositories',
+  templateUrl: './top-repositories.component.html',
+  styleUrls: ['./top-repositories.component.scss'],
+})
+export class TopRepositoriesComponent {
+  username$ = this.store.select(selectUserLoginName);
+  repos$ = this.store.select(selectTopRepos);
+  profile$?: Observable<ProfileState>;
```

Comment:
What is profile$ used for? I don't see it anywhere, but might be missing something.

- [ ] Is profile$ used anywhere? If not, we can remove that